services/QuillLMS/spec/models/segment_integration/user_spec.rb

```diff
@@ -101,6 +101,8 @@
     let(:teacher_info) { create(:teacher_info, user: teacher) }
```

Comment:
What about using the teacher_info from the teacher:

`let(:teacher_info) { teacher.teacher_info }`

then you could get rid of lines 104-105